src/app/public/modules/inline-delete/inline-delete.component.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  AnimationEvent
+} from '@angular/animations';
+
+import {
+  ChangeDetectorRef,
+  Component,
+  EventEmitter,
+  Input,
+  Output
+} from '@angular/core';
+
+import {
+  skyAnimationEmerge
+} from '@skyux/animations';
+
+import {
+  SkyInlineDeleteType
+ } from './inline-delete-type';
+
+/**
+ * Auto-incrementing integer used to generate unique ids for inline delete components.
+ */
+let nextId = 0;
+
+@Component({
+  selector: 'sky-inline-delete',
+  styleUrls: ['./inline-delete.component.scss'],
+  templateUrl: './inline-delete.component.html',
+  animations: [
+    skyAnimationEmerge
+  ]
+})
+export class SkyInlineDeleteComponent {
+
+  @Input()
+  public pending: boolean = false;
```

Comment:
Should we be proactive here and add the jsdoc descriptions for our public API properties & events?

Reply:
Uhhhhh maybe? I haven't done any new docs work so I'm not sure how those work lol.

Reply:
You just write your description above the property with the jsDoc comments:
```
/**
 * This is a function.
 *
 * @param {string} n - A string param
 */
```

Reply:
Done